### PR TITLE
Updated meta tags for SEO and link pasting reasons

### PR DESCRIPTION
**Pull request by j256**
This is somewhat a copy of https://github.com/livgust/macovidvaccines.com/pull/38 with some tweaks.  I'm happy to make the changes on develop if that is going to be merged into master soon.

### CODE REVIEW

**Comment by harcod** [comment]
I think it is technically more accurate to use "**Covid-19**" or "**COVID-19**" rather than "Covid 19" (without the hyphen)

**Comment by j256** [comment]
Why do we need to have our domain name in the title?  That seems unnecessary.  Can you find any other site that does that?  I'm fine with the hyphen.

**Comment by j256** [comment]
Maybe "Covid Vaccine Appointments for Massachusetts" as a title if you are worried about truncation? I'm not so worried about how it looks in a tab because most people done use them.

**Comment by harcod** [comment]
OK. I see your point about the domain name in the title.  How about a compromise?
> Massachusetts Covid Vaccine Appointments | MACovidVaccines.com

**Comment by j256** [comment]
Yeah that shouldn't affect anything.  Good compromise.

**Comment by harcod** [comment]
Looks good to me!

**Review by harcod — APPROVED** [review]
Tested on local build.  All images present and spelling checked out.

File: public/index.html [hunk 1]
```diff
@@ -6,17 +6,39 @@
...
+    <meta name="twitter:url" content="https://www.macovidvaccines.com/" />
+    <meta name="twitter:title" content="Covid-19 Vaccine Appointments in Massachusetts | MACovidVaccines.com" />
+    <meta name="twitter:description" content="Make an appointment to get your Massachusetts COVID vaccine" />
+    <meta name="twitter:image" content="https://www.macovidvaccines.com/logo192.png" />
+
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
-      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+      users mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
```
Comment (livgust) [inline public/index.html:41]: that apostrophe is correct!

File: public/index.html [hunk 2]
```diff
@@ -28,7 +50,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>MA Covid Vaccines</title>
+    <title>Covid-19 Vaccine Appointments in Massachusetts</title>
```
Comment (livgust) [inline public/index.html:53]: The longest I'm willing to make this is "Massachusetts COVID Vaccines" - we don't want this too long for usability purposes

File: public/index.html [hunk 3]
```diff
@@ -6,17 +6,39 @@
...
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+      content="Make an appointment to get your Massachusetts COVID vaccine"
+      />
+
+    <meta name="keywords"
+          content="massachusetts, mass, ma, covid, covid-19, vaccine, vaccines, vax, search, sign up, zipcode"
+          />
+    <meta name="robots" content="index, follow" />
+    <meta name="language" content="English" />
+    <meta name="title" content="Covid-19 Vaccine Appointments in Massachusetts | MACovidVaccines.com" />
```
Comment (livgust) [inline public/index.html:22]: Can we change all instances of "Covid" to "COVID" please? Hope you don't mind throwing that in here as well

Comment (livgust) [reply public/index.html:22]: (Except MACovidVaccines)

File: public/index.html [hunk 4]
```diff
@@ -6,17 +6,39 @@
...
+
+    <!-- Open Graph / Facebook -->
+    <meta name="og:type" content="website" />
+    <meta name="og:url" content="https://www.macovidvaccines.com/" />
+    <meta name="og:title" content="Covid-19 Appointments in Massachusetts | MACovidVaccines.com" />
```
Comment (livgust) [inline public/index.html:27]: missing "Vaccine" before "Appointments"

File: public/index.html [hunk 5]
```diff
@@ -6,17 +6,39 @@
...
+    <meta name="og:description" content="Make an appointment to get your COVID vaccine in Massachusetts" />
+    <meta name="og:image" content="https://www.macovidvaccines.com/logo192.png" />
+
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:url" content="https://www.macovidvaccines.com/" />
+    <meta name="twitter:title" content="Covid-19 Vaccine Appointments in Massachusetts | MACovidVaccines.com" />
+    <meta name="twitter:description" content="Make an appointment to get your Massachusetts COVID vaccine" />
```
Comment (livgust) [inline public/index.html:35]: Make an appointment to get your COVID vaccine in Massachusetts (for consistency)

File: public/index.html [hunk 6]
```diff
@@ -6,17 +6,39 @@
...
-    />
+      />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Make an appointment to get your COVID vaccine"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+      content="Make an appointment to get your Massachusetts COVID vaccine"
```
Comment (livgust) [inline public/index.html:14]: Make an appointment to get your COVID vaccine in Massachusetts (for consistency)

**Review by livgust — CHANGES_REQUESTED** [review]
Thanks for doing this! Few nitpicky comments

File: public/index.html (hunk 1 above)
Comment (j256) [reply public/index.html:41]: Sorry it was screwing up my editor's rendering.  :-)

File: public/index.html (hunk 2 above)
Comment (j256) [reply public/index.html:53]: Why?  When is this title used aside from SEO?  When do people see it aside from a truncated tab which is why we moved the key words up front.

File: public/index.html (hunk 3 above)
Comment (j256) [reply public/index.html]: Fixed

File: public/index.html (hunk 6 above)
Comment (j256) [reply public/index.html]: Fixed

File: public/index.html (hunk 4 above)
Comment (j256) [reply public/index.html]: Fixed

File: public/index.html (hunk 5 above)
Comment (j256) [reply public/index.html]: Fixed

File: public/index.html (hunk 2 above)
Comment (j256) [reply public/index.html]: image

This is what it looks like in Chrome for example.

**Review by livgust — APPROVED** [review]

File: public/index.html (hunk 2 above)
Comment (livgust) [reply public/index.html]: Alright, I'm good given your argument on moving key words up front! Approving.

**Pull request merged**